pkg/systemlogmonitor/log_monitor.go

```diff
@@ -165,7 +188,21 @@ func (l *logMonitor) generateStatus(logs []*logtypes.Log, rule systemlogtypes.Ru
 				break
 			}
 		}
+		if *l.config.EnableMetricsReporting {
```

Comment:
Put this into the if clause on line 176.

Reply:
Done.